docs/image.rst

```diff
@@ -0,0 +1,126 @@
+Image
+*****
+
+.. py:module:: microbit
+
+Classes
+=======
+
+.. py:class::
+    Image(string)
+    Image(width=None, height=None, buffer=None)
+
+    If ``string`` is used, it has to consist of digits 0-9 arranged into
+    lines, describing the image, for example::
+
+        image = Image("""\
+        90009
+        09090
+        00900
+        09090
+        90009""")
```

Comment:
Are the spaces ignored? If they are ignored, this could be formatted much nicer.